client/my-sites/plans-features-main/components/plan-notice.tsx

```diff
@@ -83,14 +84,21 @@ export default function PlanNotice( {
 				status="is-success"
 			>
 				{ translate(
-					'You have {{b}}%(amountInCurrency)s{{/b}} of pro-rated credits available from your current plan. ' +
-						'Apply those credits towards an upgrade before they expire!',
+					'We’ve applied the {{b}}%(amountInCurrency)s{{/b}} {{a}}upgrade credit{{/a}} from your current plan as a deduction to your new plan, below. This remaining credit will be applied at checkout if you upgrade today!',
```

Comment:
`We’ve` to `We have` maybe? a little less informal 🤔

Reply:
There are a number of examples where the tone WordPress uses is more informal -- do we typically prefer more formal language for our plans pricing optimizations?

<img width="395" alt="Screenshot 2023-05-10 at 7 29 02 AM" src="https://github.com/Automattic/wp-calypso/assets/5414230/c92c07f2-455d-42e6-881b-adde401d6732">

<img width="484" alt="Screenshot 2023-05-10 at 7 32 39 AM" src="https://github.com/Automattic/wp-calypso/assets/5414230/e4306abf-97fd-4e28-a3fc-86d63c12a046">

<img width="472" alt="Screenshot 2023-05-10 at 7 31 52 AM" src="https://github.com/Automattic/wp-calypso/assets/5414230/a4dacc8d-e0e0-4d8b-abb9-a8a0717c7ca3">

Reply:
Yes I think the objective here was to be conversational. Can you weigh in here @tanjoymor  ? 🙂

Reply:
Informal is fine in any case :-)